Makefile

```diff
@@ -568,8 +568,17 @@ vendor-in-container:
 package:  ## Build rpm packages
 	## TODO(ssbarnea): make version number predictable, it should not change
 	## on each execution, producing duplicates.
-	rm -f  ~/rpmbuild/RPMS/x86_64/* ~/rpmbuild/RPMS/noarch/*
+	rm -rf build/* *.src.rpm ~/rpmbuild/RPMS/*/*
 	./contrib/build_rpm.sh
 
-package-install: package  ## Install rpm packages
-	sudo ${PKG_MANAGER} -y install --allowerasing ${HOME}/rpmbuild/RPMS/*/*.rpm
+package-install: package  ## Install rpm packages via a local podman.repo
+	# We use the repository approach with max priority and without disabling
+	# system packages in order to detect if our packages are
+	# creating any conflicts.
```

Comment:
Suggestion: these comments will also display on stdout when the `package-install` recipe is executing.  Not a huge problem but does add a bit to output clutter.  Would the comment be just as useful for maintainers if placed above the target?